tasks/resolve_reference.json

```diff
@@ -17,6 +17,13 @@
     "credentials": {
       "type": "Optional[String]"
     },
+    "access_key_id": {
+      "type": "Optional[String]"
+    },
+    "secret_access_key": {
+      "type": "Optional[String]",
+      "sensitive": true
+    },
```

Comment:
Can you add these to the [README](https://github.com/puppetlabs/puppetlabs-aws_inventory/blob/main/README.md#usage) as available options?